src/ess/imaging/data.py

```diff
@@ -22,6 +22,8 @@ def _make_pooch():
             'siemens_star.tiff': 'md5:0ba27c2daf745338959f5156a3b0a2c0',
             'resolving_power_test_target.tiff': 'md5:a5d414603797f4cc02fe7b2ae4d7aa48',
             'tbl-orca-focussing.hdf.zip': 'md5:f365acd9ea45dd205c0b9398d163cfa4',
+            "siemens-star-measured.h5": "md5:8e333d36c7c102f474b2b66cb785f5e8",
+            "siemens-star-openbeam.h5": "md5:ee429b2c247aeaafb0ef3ca4171f2e6a",
```

Comment:
I think we have a similar issue with some of the other files in here, so it's not specific to these changes: we should probably add descriptions on how these images were created (link to a script we used if we have that, or link to where we got the image from).